deisctl/units/deis-logger.service

```diff
@@ -15,3 +15,7 @@ RestartSec=5
 
 [Install]
 WantedBy=multi-user.target
+
+[X-Fleet]
+MachineMetadata="deisRole=general"
+MachineMetadata="deisRole=control"
```

Comment:
I tested the syntax you suggested and it does not have the same effect.  That is to say, it is not interpreted as a logical or.  In the example you gave, "control,general" is evaluated as the machine's `deisRole`.